com.unity.render-pipelines.core/Editor/Lighting/Shadow/ShadowCascadeGUI.cs

```diff
@@ -0,0 +1,516 @@
+
+                cascadeWidths[i] = pixelPerfectCascadeWidth;
+                sumOfCascadeWidthsWithoutLast += cascadeWidths[i];
+
+                startX = endX;
+            }
+            cascadeWidths[cascades.Length - 1] = widthForCascades - sumOfCascadeWidthsWithoutLast;
+
+            float currentX = usableRect.x;
+            for (int i = 0; i < cascades.Length; ++i)
+            {
+                ref var cascade = ref cascades[i];
+                var cascadeWidth = cascadeWidths[i];
+
+                bool isLastCascade = (i == cascades.Length - 1);
+
+                // Split cascade into cascade without border and border
+                float borderValue;
+                float cascadeValue;
+                float borderWidth;
+                float cascadeWithoutBorderWidth;
+                if (cascade.borderHandleState != HandleState.Hidden)
+                {
+                    borderValue = cascade.size * cascade.borderSize;
+                    cascadeValue = cascade.size - borderValue;
+                    var cascadeWidthWithoutPartition = cascadeWidth;
+                    cascadeWithoutBorderWidth = Mathf.Round(cascadeWidthWithoutPartition * (1 - cascade.borderSize) * EditorGUIUtility.pixelsPerPoint) / EditorGUIUtility.pixelsPerPoint;
+                    borderWidth = cascadeWidth - cascadeWithoutBorderWidth;
+                }
+                else
+                {
+                    borderValue = 0;
+                    cascadeValue = cascade.size;
+                    borderWidth = 0;
+                    cascadeWithoutBorderWidth = cascadeWidth;
+                }
+
+                // Draw cascade
+                var cascadeRect = new Rect(currentX, usableRect.y, cascadeWithoutBorderWidth, usableRect.height);
+                currentX += DrawBoxGUI(cascadeRect, kCascadeColors[i]);
+
+                // Draw cascade text
+                float cascadeValueForText = useMetric ? cascadeValue * baseMetric : cascadeValue * 100;
+                string cascadeText = $"{i}\n{cascadeValueForText:F1}{formatSymbol}";
+                DrawLabelGUI(cascadeRect, cascadeText, Color.white);
+
+                if (cascade.borderHandleState != HandleState.Hidden)
+                {
+                    // As we are rounding everything against pixel per point and subtracting from total it might result in fractions for the last cascade border
+                    if (isLastCascade && cascade.borderSize == 0.0)
+                        borderWidth = 0;
+
+                    // Draw border snatch handle
+                    var borderPartitionHandleRect = new Rect(
+                        currentX - 6 - partitionHalfWidth,
+                        usableRect.y + usableRect.height - 1,
+                        12,
+                        18);
+                    var enabled = cascade.borderHandleState == HandleState.Enabled;
+                    var borderPartitionColor = enabled ? kCascadeColors[i] : kDisabledColor;;
```

Comment:
2 :: at the end of this line
Nitpicky I know :)

Reply:
Fixed it, we actually have quite a lot of them in code base :D. We should probably clean it up in some quality push